README.md

```diff
@@ -529,6 +529,49 @@ server.listen(443, () => {
 
 </details>
 
+<details>
+<summary>Server usage with <a href="https://github.com/apollographql/apollo-server/tree/main/packages/apollo-server-express">Apollo Server Express</a></summary>
+
+```typescript
+import https from 'https';
+import express from 'express';
+import { ApolloServer } from 'apollo-server-express';
+import { createServer } from 'graphql-ws';
+import { execute, subscribe } from 'graphql';
+import { schema } from 'my-graphql-schema';
+
+// create express
+const app = express();
+
+// create apollo server
+const apolloServer = new ApolloServer({
+    schema,
+    subscriptions: false // this is important
```

Comment:
I guess because it disables `subscriptions-transport-ws`? If so, would be nice to state that.Just to reduce confusion.

```suggestion
    subscriptions: false, // disable subscriptions-transport-ws
```